netbox/dcim/choices.py

```diff
@@ -278,6 +278,15 @@ class PowerPortTypeChoices(ChoiceSet):
     TYPE_ITA_M = 'ita-m'
     TYPE_ITA_N = 'ita-n'
     TYPE_ITA_O = 'ita-o'
+    # USB
+    TYPE_USB_A = 'usb-a'
+    TYPE_USB_MINIA = 'usb-mini-a'
+    TYPE_USB_MICROA = 'usb-micro-a'
+    TYPE_USB_B = 'usb-b'
+    TYPE_USB_MINIB = 'usb-mini-b'
+    TYPE_USB_MICROB = 'usb-micro-b'
+    TYPE_USB_3_MICROB = 'usb-3-micro-b'
```

Comment:
Is it worth calling out USB 3.0 micro B as its own form factor? Seems like a very rare connector that might just introduce confusion. (If it _is_ worth adding, we probably need to add a USB 3.0 variant for the type B plug as well.)

Reply:
The usb3.0 microB is pretty prevelant on external HDD's with USB3.0 afaik. I've seen them there a few times. Doesn't hurt to include it, right? I'll add the usb3.0 type B variant then as well :+1: